matlab_simulation/code v1.0/ME597-8-NLP/Bz_fun.m

```diff
@@ -0,0 +1,9 @@
+function Sp = Bz_fun(g)
+%BZ_FUN
+%    SP = BZ_FUN(G)
+
+%    This function was generated by the Symbolic Math Toolbox version 6.1.

```

Comment:
Why is this curve hard coded? If you want to use Bezier curves, make it possible to generate them on the fly for each problem instance.  I'm fine with just using sine or cosine trajectories, though, as Bezier would be another thing to explain in class.